specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/2016-12-01/backupManagement.json

```diff
@@ -1107,101 +1104,6 @@
         "deprecated": false
       }
     },
-    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/backupJobs": {
```

Comment:
@jhendrixMSFT I don't get it. What other operations are removed? There are just 2 operations removed from backupManagement.json and both are added to jobs.json with new api version.